scripts/mail-all-activity.py

```diff
@@ -1,181 +0,0 @@
-#!/usr/bin/env python
```

Comment:
OK, if the email settings are documented somewhere so that we can find 
them next time, then +1 to merge.